src/adaptors/monolith-market/index.js

```
@@ -0,0 +1,190 @@
+    const underlying = coins[marketIndex];
+    const vaultSymbol = vaultSymbols[marketIndex];
+    const coinPriceUsd = pricesByAddress[underlying];
+    const totalSupplyUsd = coinPriceUsd * Number(vaultAssets[marketIndex]) / 1e18
+    const stakingApr = Number(rates[marketIndex][1]) / 1e16;
+
+    return {
+      pool: `monolith-market-savings-${vaults[marketIndex]}`,
+      chain: 'Ethereum',
+      project: PROJECT,
+      symbol: vaultSymbol,
+      tvlUsd: totalSupplyUsd,
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)